src/app/GitUI/CommandsDialogs/BrowseDialog/FormOpenDirectory.cs

```diff
@@ -21,7 +21,7 @@ public FormOpenDirectory(IGitModule? currentModule)
 
             IList<Repository> repositoryHistory = ThreadHelper.JoinableTaskFactory.Run(RepositoryHistoryManager.Locals.LoadRecentHistoryAsync);
             _NO_TRANSLATE_Directory.DataSource = GetDirectories(currentModule, repositoryHistory);
-            _NO_TRANSLATE_Directory.ResizeDropDownWidth(AppSettings.BranchDropDownMinWidth, AppSettings.BranchDropDownMaxWidth);
+            _NO_TRANSLATE_Directory.ResizeDropDownWidth(DpiUtil.Scale(AppSettings.BranchDropDownMinWidth), DpiUtil.Scale(AppSettings.BranchDropDownMaxWidth));
```

Comment:
We are updating all callsites. Any reason not to update the implementation to scale the passed arguments?

Reply:
While it would be more convenient and produce less diffs I think that semantically this is correct - this is a straightforward utility method and the caller has control and responsibility to understand what is being done. Unless we'd add `ResizeDropDownWidthDpiAware`..
Anyhow, I see your point. I personally would lean towards semantic correctness (which I guess could also be debated :) ) and leave it the way it is at the same time this is not the hill I'm ready to die on so if you'll tell me to change this, I will.

Reply:
Can you think of a scenario where we'd want to resize a dropdown without scaling for the current dpi? If not, then I can't see any reason why we shouldn't be doing scaling implicitly - i.e., not rely on a caller to remember to scale before resizing.

Reply:
One scenario came up, it's when you want to adjust the width based on some other control width which is already scaled. You'd need to de-scale it which is not pretty. I solved it by introducing an optional parameter `bool dpiScaleBounds = true` and this one place which didn't require scaling sets it to `false`